relay-conventions/src/consts.rs

```diff
@@ -0,0 +1,68 @@
+pub const BROWSER_NAME: &str = "sentry.browser.name";
+pub const BROWSER_VERSION: &str = "sentry.browser.version";
+pub const DB_QUERY_TEXT: &str = "db.query.text";
+pub const DB_STATEMENT: &str = "db.statement";
+pub const DB_SYSTEM_NAME: &str = "db.system.name";
+pub const DESCRIPTION: &str = "sentry.description";
+pub const FAAS_TRIGGER: &str = "faas.trigger";
+pub const GEN_AI_SYSTEM: &str = "gen_ai.system";
+pub const GRAPHQL_OPERATION: &str = "sentry.graphql.operation";
+pub const HTTP_PREFETCH: &str = "sentry.http.prefetch";
+pub const HTTP_REQUEST_METHOD: &str = "http.request.method";
+pub const HTTP_ROUTE: &str = "http.route";
+pub const HTTP_TARGET: &str = "http.target";
+pub const MESSAGING_SYSTEM: &str = "messaging.system";
+pub const OBSERVED_TIMESTAMP_NANOS: &str = "sentry.observed_timestamp_nanos";
+pub const OP: &str = "sentry.op";
+pub const PROFILE_ID: &str = "sentry.profile_id";
+pub const RELEASE: &str = "sentry.release";
+pub const RPC_SERVICE: &str = "rpc.service";
+pub const SEGMENT_ID: &str = "sentry.segment.id";
+pub const SEGMENT_NAME: &str = "sentry.segment.name";
+pub const STATUS_MESSAGE: &str = "sentry.status.message";
+pub const URL_FULL: &str = "url.full";
+pub const URL_PATH: &str = "url.path";
+pub const USER_GEO_CITY: &str = "user.geo.city";
+pub const USER_GEO_COUNTRY_CODE: &str = "user.geo.country_code";
+pub const USER_GEO_REGION: &str = "user.geo.region";
+pub const USER_GEO_SUBDIVISION: &str = "user.geo.subdivision";
```

Comment:
Would it make sense to put all of these in a struct, s.t. in the test case you can destructure the struct so new additions are forced to add a check to the test case?